### PR TITLE
Move to stable channel + bump to 4.13

### PR DESCRIPTION
**Pull request by locriandev**
Ref. https://issues.redhat.com/browse/ART-5087

### CODE REVIEW

**Comment by locriandev** [comment]
Test build: 
- https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2263092
- https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2263153

File: manifests/stable/cluster-kube-descheduler-operator.v4.13.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -28,9 +28,9 @@ metadata:
         }
       ]
     certified: "false"
-    containerImage: quay.io/openshift/origin-cluster-kube-descheduler-operator:4.12
+    containerImage: quay.io/openshift/origin-cluster-kube-descheduler-operator:4.13
     createdAt: 2021/10/13
-    olm.skipRange: ">=4.9.0 <4.12.0"
+    olm.skipRange: ">=4.9.0 <4.13.0"
```
Comment (ingvagabund) [inline manifests/stable/cluster-kube-descheduler-operator.v4.13.0.clusterserviceversion.yaml:33]: s/4.9.0/4.10.0

File: manifests/art.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 updates:
-  - file: "{MAJOR}.{MINOR}/cluster-kube-descheduler-operator.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
+  - file: "stable/cluster-kube-descheduler-operator.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
```
Comment (ingvagabund) [inline manifests/art.yaml:2]: Is renaming the `{MAJOR}.{MINOR}` folder into `stable` necessary? I see the benefit of reducing the number of changes when bumping the version. Nevertheless, is this a pattern every OLM managed operator is expected to follow?

Comment (locriandev) [reply manifests/art.yaml:2]: According https://issues.redhat.com/browse/ART-3107, all operators should adopt the `stable` pattern

**Comment by ingvagabund** [comment]
@locriandev would you please set the PR against the master branch?

**Comment by ingvagabund** [comment]
https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/manifests/art.yaml#L10-L11 needs to be updated as well from 4.9 to 4.10.

**Comment by ingvagabund** [comment]
Just the last change under art.yaml, the rest looks good
/approve

**Comment by locriandev** [comment]
> Just the last change under art.yaml, the rest looks good /approve

Done in https://github.com/openshift/cluster-kube-descheduler-operator/pull/279/commits/61810bda962947390e09bcb924e3f00ad80bbad3

**Pull request merged**